crates/astria-composer/src/collectors/grpc.rs

```diff
@@ -24,7 +21,10 @@ use tonic::{
     Status,
 };
 
-use crate::executor;
+use crate::{
+    collectors::geth::EXECUTOR_SEND_TIMEOUT,
```

Comment:
Remove this. The grpc collector schould be entirely independent from what's going on in geth